tests/unit/migrations/range.test.ts

```diff
@@ -0,0 +1,50 @@
+import { describe, expect, it } from "@jest/globals";
+
+import { parseRange, validateRange } from "../../../src/modules/migrations/utils/rangeUtils";
+
+describe("parseRange", () => {
+  it("should return an error for bad range format", () => {
+    expect(parseRange("T2023-01-01")).toEqual({ err: "Bad range format" });
+    expect(parseRange("T202:")).toEqual({ err: "T202 is not a valid order format." });
+    expect(parseRange("Tfdsa:")).toEqual({ err: "Tfdsa is not a valid order format." });
+    expect(parseRange("T2024:T202")).toEqual({ err: "T202 is not a valid order format." });
+    expect(parseRange("T2024-0:T202")).toEqual({ err: "T2024-0 is not a valid order format." });
+    expect(parseRange("T2024-04:2024-05")).toEqual({ err: "2024-05 is not a valid order format." });
+    expect(parseRange("T2023-01-01:2023-01-01:2023-01-01")).toEqual({ err: "Bad range format" });
+  });
+
+  it("should parse ranges correctly", () => {
```

Comment:
I don't see any range using hours, minutes or seconds. We should test that as well.